.github/scripts/build-fleet-binaries.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Description: build fleet binary and image with debug flags
+
+set -euxo pipefail
+
+if [ ! -d ./cmd/fleetcontroller ]; then
+  echo "please change the current directory to the fleet repo checkout"
+  exit 1
+fi
+
+export GOARCH="${GOARCH:-amd64}"
+export CGO_ENABLED=0
+
+# re-generate code
+if ! git diff --quiet HEAD origin/master --  pkg/apis/fleet.cattle.io/v1alpha1; then
+  go generate
+fi
```

Comment:
```suggestion
```

I'd also remove this. We have a check in another action, to protect against a missing generate, I think. In any case, the CI shouldn't generate code, but use the already generated code.

Reply:
Correct, [this workflow](https://github.com/rancher/fleet/blob/master/.github/workflows/check-changes.yml) is called against pull requests and already checks for this. Removed, thanks.